From d1a9562c486537aa51b0a877045f353793deb771 Mon Sep 17 00:00:00 2001
From: Ondrej Jirman <megous@megous.com>
Date: Thu, 25 Jun 2020 01:57:27 +0200
Subject: [PATCH 340/446] ARM: dts: sun5i-a13-pocketbook-touch-lux-3: Add RTC
 clock-cells

This stop the driver from complaining in dmesg.

Signed-off-by: Ondrej Jirman <megous@megous.com>
---
 arch/arm/boot/dts/sun5i-a13-pocketbook-touch-lux-3.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/sun5i-a13-pocketbook-touch-lux-3.dts b/arch/arm/boot/dts/sun5i-a13-pocketbook-touch-lux-3.dts
index db28ab621..ee81db68e 100644
--- a/arch/arm/boot/dts/sun5i-a13-pocketbook-touch-lux-3.dts
+++ b/arch/arm/boot/dts/sun5i-a13-pocketbook-touch-lux-3.dts
@@ -132,6 +132,7 @@ &i2c1 {
 	pcf8563: rtc@51 {
 		compatible = "nxp,pcf8563";
 		reg = <0x51>;
+		#clock-cells = <0>;
 	};
 
 	// hacky PMIC driver for eInk display
-- 
2.31.1

